prow/gerrit/adapter/adapter.go

```diff
@@ -585,6 +585,32 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 			logger.WithField("lastUpdate", lastUpdate).Warnf("lastUpdate not found, falling back to now")
 		}
 
+		if change.Revisions != nil {
```

Comment:
the purpose of this PR is to skip the inrepoconfig processing, this check will need to be moved to before line 537 to work as expected

Reply:
Done.